configs/eslint-common-react.json

```diff
@@ -1,10 +1,8 @@
 {
     "extends": [
-        "./eslint-common.json",
         "airbnb",
```

Comment:
That was the main problem encountered by @leon-good-life with his PR - https://github.com/cloudify-cosmo/cloudify-stage/pull/1796 . Order matters in ESLint, and `airbnb` was overwriting changes introduced to `eslint-common.json`.